config/settings_schema.json

```diff
@@ -590,7 +590,7 @@
         "type": "range",
         "id": "card_shadow_blur",
         "min": 0,
-        "max": 100,
+        "max": 40,
```

Comment:
I think this should be 30 a to match the others. Or the others should be 40 as well 😅

Reply:
I made a scale change, `Variant pills` at 20 because they are so tiny, `Buttons` and `Inputs` at 30 and larger components like `Cards` etc at 40. Very subjective! 😅

Reply:
cc. @kmeleta How do you both feel about this curated difference for different components for now?

Reply:
I'm good with that. The same values across the board would make me feel warm and fuzzy, but if it doesn't artistically make sense to do that I think it's sensible to assign a small, med, and large element max.

Reply:
Sounds good! I will resolve all those comments that are about the same thing then 👍